azerothcore/azerothcore-wotlk
# DB/Core: Heroes of old Blizzlike and Fix

##### CHANGES PROPOSED:

-  Make Heroes of Old blizzlike
proof: https://www.youtube.com/watch?v=p52Jpb1Xu28
In this video -> https://www.youtube.com/watch?v=1dSz4a-g3yE it can be seen when he accepts the quest that the blizzlike uses phases for this quests.


###### ISSUES ADDRESSED:


Closes 


##### TESTS PERFORMED:




##### HOW TO TEST THE CHANGES:


.go creature 35237
.quest add 2681
.quest complete 2681 and reward it
Take the quest "Hero of old" now, speak with the fallen hero of the horde and complete the quest.( The NPC should be spawned across the road) 
If the player gets disconected or something is messing up his phase, he can click " Hero, i have a urgent business with Thund Splithoof" and the correct phase is set.


##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
